packages/discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -902,6 +935,26 @@ def fetch_existing_entities(session: Session, entities_to_fetch: EntitiesToFetch
             app_json["address"].lower(): app_json for _, app_json in developer_apps
         }
 
+    # DASHBOARD WALLETS
+    if entities_to_fetch["DashboardWalletUser"]:
+        dashboard_wallets: List[Tuple[DashboardWalletUser, dict]] = (
+            session.query(
+                DashboardWalletUser,
+                literal_column(f"row_to_json({DashboardWalletUser.__tablename__})"),
```

Comment:
i dont have much context, but this this an optimization for the column vs what i imagine would be a simpler value here?

Reply:
i think i tried to do this in a better way but couldn't figure it out. this basically fetches relevant wallets and gives you the entire row + the row in json form for revert cases